

confirm_act : True
debug_print : True

moveit_group : "left_arm"

planning_id : "RRTConnectkConfigDefault"
planning_attempts : 1
position_tolerance : 0.01  # 1cm
orientation_tolerance : 0.02  # 0.57 * 2 deg
planning_time : 0.1        # 0.1s
max_vel_scale_factor : 0.5          # move_group
jump_threshold : 2.0  
trajectory_velocity_scaling : 0.5  # trajectory processing
max_plan_steps : 20  
max_cartesion_plan_steps : 100 


# Get current joint state from the topic
kinova_driver_joint_state_topic : "j2n6s300_driver/out/joint_state"
# Get current end effector pose from the topic
kinova_driver_tool_pose_topic : "j2n6s300_driver/out/tool_pose"
joint_states_topic : "joint_states"

moveit_traj_action_topic : "/l_arm_controller/follow_joint_trajectory"

moveit_traj_arm_base_frame : "base_link"

use_arm : "left"

x : 0.
y : 0.
z : 0.1
num1 : 200
num2 : 1